### ISSUE TITLE
Dockerizing counter example

### ISSUE THREAD

**Issue by egekorkan** [eclipse-thingweb/node-wot#1330]
In order to host it in the test-things infrastructure, counter needs to be dockerized. This is how the old test thing looks like now.

I am guessing that removing the CLI dependency will be good and build the WoT object in the code. Any opinions?

**Comment by sailalithkanumuri8** [comment]
I would agree, as I feel that building it directly in code is generally better and cleaner. Removing the CLI dependency now will reduce dependency management and will also decrease startup costs.

**Comment by egekorkan** [comment]
https://github.com/eclipse-thingweb/test-things/blob/main/things/data-schema-thing/http/ts/src/main.ts should be used as a template for the counter. Adding dependencies, docker image creation, publishing the image (together) etc.

**Comment by relu91** [comment]
If we remove the CLI, do we have another example of a Dockerized application that uses the `wot-servient` command? Should it be simply a documentation page?
